dist/windows/installer-translations/afrikaans.nsi

```diff
@@ -14,6 +14,8 @@ LangString inst_torrent ${LANG_AFRIKAANS} "Open .torrent files with qBittorrent"
 LangString inst_magnet ${LANG_AFRIKAANS} "Open magnet links with qBittorrent"
 ;LangString inst_firewall ${LANG_ENGLISH} "Add Windows Firewall rule"
 LangString inst_firewall ${LANG_AFRIKAANS} "Add Windows Firewall rule"
+;LangString inst_pathlimit ${LANG_ENGLISH} "Disable MAX_PATH length limit (requires Windows 10 1607 or later)"
+LangString inst_pathlimit ${LANG_AFRIKAANS} "Disable MAX_PATH length limit (requires Windows 10 1607 or later)"
```

Comment:
This message is for regular users. I wouldn't meantion any Windows internals (MAX_PATH constant). Why not just write it as "Disable path length limit (requires Windows 10 1607 or later)"?

Reply:
I think we should make it clear that this is a Windows limitation. In this case, I think it is acceptable to mention `MAX_PATH` to end-users, since this is a very common Windows issue and this is how it is known by.

I have in mind something based on what the Python installer does (https://github.com/qbittorrent/qBittorrent/issues/6645#issuecomment-509729380):

`Disable Windows path length limit (260 character MAX_PATH limitation, requires Windows 10 1607 or later)`

Too long? If so, this is also acceptable IMO:

`Disable Windows path length limit (requires Windows 10 1607 or later)`

which is similar to what @glassez proposes, but clearly states this is a Wndows issue.

Otherwise, if it is possible to add smaller subtext, we should basically copy-paste the Python installer text.

Reply:
>"Disable Windows path length limit (260 character MAX_PATH limitation, requires Windows 10 1607 or later)"

I would choose this one, better to be detailed than some obscure description that requires further googling or digging into source code.

Reply:
> 
> 
> > "Disable Windows path length limit (260 character MAX_PATH limitation, requires Windows 10 1607 or later)"
> 
> I would choose this one, better to be detailed than some obscure description that requires further googling or digging into source code.

:+1:

Reply:
glassez I just pasted the one suggested by chocobo1. Initially I didn’t use that.
Currently I don’t have a PC for maybe next 2-3 wk. So unfortunately I can’t update the PR till then.

Reply:
I updated them.

Reply:
Thanks a lot! Sorry for the trouble.